src/ludamus/templates/chronology/enroll_select.html

```
@@ -125,7 +125,7 @@ <h3 class="font-semibold mb-2 text-foreground-secondary">{% translate "About thi
                     {# feed enroll-preview.ts, which projects each ticked row to #}
                     {# "gets a seat" / "joins the waiting list" using the same order #}
                     {# and seat accounting as the server's routing. #}
-                    <div class="rounded-2xl border border-border bg-bg-secondary overflow-hidden shadow-xl"
+                    <div class="rounded-2xl bg-bg-secondary overflow-hidden smooth-shadow-ring-xl"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)